app/models/lookup_keys/lookup_key.rb

```diff
@@ -232,12 +206,6 @@ def load_yaml_or_json(value)
     end
   end
 
-  def ensure_type
-    if puppetclass_id.present? and is_param?
-      self.errors.add(:base, _('Global variable or class Parameter, not both'))
```

Comment:
This error was not only confusing but I think it only shows up in the logs, not the UI on the develop branch now. Great to see it going away.